packages/vscode-ws-jsonrpc/src/disposable.ts

```diff
@@ -1,33 +1,33 @@
 /* --------------------------------------------------------------------------------------------
- * Copyright (c) 2018 TypeFox GmbH (http://www.typefox.io). All rights reserved.
+ * Copyright (c) 2018-2022 TypeFox GmbH (http://www.typefox.io). All rights reserved.
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
+
 import { Disposable } from 'vscode-jsonrpc/lib/common/disposable';
 
 export {
     Disposable
-}
+};
 
 export class DisposableCollection implements Disposable {
     protected readonly disposables: Disposable[] = [];
 
-    dispose(): void {
+    dispose (): void {
         while (this.disposables.length !== 0) {
             this.disposables.pop()!.dispose();
         }
     }
 
-    push(disposable: Disposable): Disposable {
+    push (disposable: Disposable): Disposable {
         const disposables = this.disposables;
         disposables.push(disposable);
         return {
-            dispose(): void {
+            dispose (): void {
                 const index = disposables.indexOf(disposable);
                 if (index !== -1) {
                     disposables.splice(index, 1);
                 }
             }
-        }
+        };
     }
-
-}
\ No newline at end of file
+}
```

Comment:
(not related to this PR) Are you sure it's relevant to have the same code duplicated in the client and vscode-ws-jsonrpc?

Reply:
Not nice, but both packages are independent. Only the examples use both.

Reply:
I actually have a good idea for that.

Reply:
I forced another update. At least both `disposable.ts` are identical now.